src/main/resources/add-ons-to-index.json

```diff
@@ -783,6 +783,41 @@
         "package": "clinicalsummary"
       }
     },
+    {
+      "uid": "org.openmrs.module.commonlabtest",
+      "type": "OMOD",
+      "name": "Common Lab Test",
+      "maintainers": [
+        {
+          "name": "Owais Hussain"
+        },
+        {
+          "name": "Tahira Niazi"
+        }
+      ],
+      "links": [
+        {
+          "rel": "documentation",
+          "href": "https://wiki.openmrs.org/display/docs/Common+Lab+Module",
+          "title": "Wiki Page"
+        },
+        {
+          "rel": "documentation",
```

Comment:
I am not sure if we can have multiple documentation links. So, to be on the safer side, I would request you to remove either of the links. I would have probably removed the link to the API and added that link within the documentation of the module.